slowmovie.py

```diff
@@ -95,8 +94,8 @@ def video_info(file):
         stream = probeInfo["streams"][0]
 
         # Calculate framerate
-        r_fps = stream["r_frame_rate"]
-        fps = float(Fraction(r_fps))
+        avg_fps = stream["avg_frame_rate"]
```

Comment:
Did you test if this works with .mkv files? If not I can give it a shot to make sure. I remember the .mkv file I was working with not having a weird assortment of the fields that the .mp4 had in the probe

Reply:
I can still read the framerate from mkv's.
```
pi@pizero:~/tests $ python3 ffprobe.py test.mkv
Framerate: 23.986019736842106
```

Reply:
I've just been running them through a test script to make sure the values are there.
```python
#!/usr/bin/python3
import os, json, sys
import ffmpeg
from fractions import Fraction

if len(sys.argv) > 1:
    videoinfo = ffmpeg.probe(sys.argv[1], select_streams = "v")
    #with open('ffprobe.json', 'w') as f:
    #    json.dump(videoinfo, f)
    #print(f'Frame count: {int(videoinfo["streams"][0]["nb_frames"])}')
    print(f'Framerate: {float(Fraction(videoinfo["streams"][0]["avg_frame_rate"]))}')

```